BranchSDK/BNCServerAPI.m

```diff
@@ -24,53 +24,74 @@ + (BNCServerAPI *)sharedInstance {
     return serverAPI;
 }
 
+- (instancetype)init {
+    self = [super init];
+    if (self) {
+        self.useTrackingDomain = NO;
+        self.useEUServers = NO;
+        self.automaticallyEnableTrackingDomain = YES;
+    }
+    return self;
+}
+
 - (NSURL *)installServiceURL{
```

Comment:
@echo Can we return only NSString from these functions instead of NSURL? For now these functions are only used by `[BNCServerInterface postRequest ]` API which expects a String. We are using absoluteString in those functions calls to get string. For example - https://github.com/BranchMetrics/ios-branch-deep-linking-attribution/blob/69c84d0724133ed16ca15ff7bddc189b32d71bff/BranchSDK/BranchOpenRequest.m#L52